modules/mobile/lib/scripts/appointments_list_validation.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require 'faraday'
+require 'json'
+require 'pry'
+
+# This script was written to help us validate what appears in the web appointments indexes
+# vs the mobile appointments index. It compares the appointment ids in the web html against
+# the appointments received from the mobile appointments index. It then formats the results
+# to highlight which ids aren't present in both sources.
+
+# This script is only being used for validating v2 appointments work and will probably not be of
+# any value for long. Delete it once we're confident that v2 appointments are working correctly.
+
+# INSTRUCTIONS FOR USE:
+# 1) in the web app, use the inspector to copy the html. Do not attempt to save the html by saving
+# the web page. That approach omitted the important parts. Paste the html into files with a
+# .txt extension in a directory that contains no other txt files.
+# 2) fetch an access token for the test user
+# 3) in terminal, run:
+# bundle exec ruby modules/mobile/lib/scripts/appointments_list_validation.rb path/to/dir/with/txt/files access_token
+
+# WARNING: for some reason the useCache param does not work. You may have to break the cache via postman instead.
```

Comment:
I spent way too much time trying to figure out what was going on here. It works fine from postman and curl but not via faraday. It was very clear that the useCache param wasn't working because it would return results immediately, which shouldn't happen when bypassing the cache. I thought it was a bug with how we handle the params but the param validator actually handles it correctly. I did find another small issue with how pagination links are created and will make a ticket for that.